### ISSUE TITLE
"Illegal instruction: 4" error in analysis with [-p] or [-S] in version 2.0-rc1

### ISSUE THREAD

**Issue by edgardomortiz** [Cibiv/IQ-TREE#116]
I got the following error for two analyses on OSX v10.13.6 (High Sierra):
```
ERROR: STACK TRACE FOR DEBUGGING:
ERROR: 2   _sigtramp()
ERROR: 4   Terrace::Terrace(PhyloTree&, SuperAlignment*)
ERROR: 5   reportPhyloAnalysis(Params&, IQTree&, ModelCheckpoint&)
ERROR: 6   runPhyloAnalysis(Params&, Checkpoint*)
ERROR:
ERROR: *** IQ-TREE CRASHES WITH SIGNAL ILLEGAL INSTRUCTION
ERROR: *** For bug report please send to developers:
ERROR: ***    Log file: grasshoppers_concat.log
ERROR: ***    Alignment files (if possible)
Illegal instruction: 4
```
I am attaching the `.log` files, I could email the alignments if necessary.
Since this error seems to happen _after_ the final trees are written, could I still use these results?

Thanks for the amazing software!

Edgardo

[grasshoppers_concat.log](https://github.com/Cibiv/IQ-TREE/files/4134512/grasshoppers_concat.log)
[grasshoppers_loci.log](https://github.com/Cibiv/IQ-TREE/files/4134513/grasshoppers_loci.log)

**Comment by OlgaChern** [comment]
Dear Edgardo,

in principle, indeed the tree search is completed. It seems there is a problem with reporting the information about Phylogenetic Terraces. 
In the presence of missing data there are might be many more trees that from a mathematical point of view are equally good, i.e. have identical score. The tree search finds a tree with the highest score and then there is an additional check if there are other equally good trees. Maybe, this check was actually wrongly called in your analysis. We’ll check what caused the problem.

Cheers,
Olga

**Comment by bqminh** [comment]
@Olga, can you have a look at this problem, thanks!

@Erdgado: as a quick fix right now, please add option --no-terrace into the command line, to avoid this step.

Cheers,
Minh

**Comment by edgardomortiz** [comment]
Thank you very much!, I will do that
Edgardo

**Comment by OlgaChern** [comment]
The issue is fixed with the commit 1a0fcd7.
